src/Stepper/_tests_/index.test.js

```diff
@@ -63,11 +46,7 @@ describe("Stepper", () => {
         .prop("data-test"),
     ).toBe(dataTest);
   });
-  it("should execute onChange method", () => {
```

Comment:
Don't you wanna try the `@testing-library/react` now? This refactor doesn't have to be merged asap and it would be great to try it 🙂

Reply:
@vepor not sure how to test it with that library now